app/controllers/project_features_controller.rb

```diff
@@ -8,10 +8,12 @@ class ProjectFeaturesController < ApplicationController
   add_breadcrumb I18n.t("breadcrumbs.project_features"), :project_features_path
 
   def index
-    year, month = params[:month_year]&.split("-")
+    @year, @month = params[:month_year]&.split("-")
+    @month ||= Date.current.month
```

Comment:
không làm thế e ơi
nếu params là trống thì cần trả về toàn bộ danh sách nha

Reply:
tham khảo bên resource

Reply:
oke ạ c